fastcrypto/Cargo.toml

```diff
@@ -37,6 +37,7 @@ typenum = "1.15.0"
 # TODO: switch to https://github.com/celo-org/celo-bls-snark-rs
 # when https://github.com/celo-org/celo-bls-snark-rs/issues/228 is solved
 celo-bls = { git = "https://github.com/huitseeker/celo-bls-snark-rs", branch = "updates-2-with-parallelism-toggle", package = "bls-crypto", default-features = false }
+p256 = { version = "0.11.1", features = ["ecdsa"] }
```

Comment:
Indeed, this is unfortunately the lib to used, unless @huitseeker has seen any fork of the one we use secpk1.

Reply:
We will also need the regular verification in both ECDSAs as i/ requires for our fair benchmarks + to switch if in the future we decide that cost is more important than size.

Reply:
You could try the arithmetic generated by fiat-crypto, which is usually pretty darn fast:
https://github.com/mit-plv/fiat-crypto/tree/master/fiat-rust/src
I haven't run benches to compare to k256/p256, though.

Reply:
With the most recent commits, we are reimplementing all of ECDSA anyway, so using a different lib could make sense. We should run some benchmarks first to see if it's worth it though.